test/components/views/avatars/__snapshots__/RoomAvatar-test.tsx.snap

```diff
@@ -0,0 +1,76 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`RoomAvatar should render as expected for a DM room 1`] = `
+<div>
+  <span
+    class="mx_BaseAvatar"
+    role="presentation"
+  >
+    <span
+      aria-hidden="true"
+      class="mx_BaseAvatar_initial"
+      style="font-size: 23.400000000000002px; width: 36px; line-height: 36px;"
+    >
+      D
+    </span>
+    <img
+      alt=""
+      aria-hidden="true"
+      class="mx_BaseAvatar_image"
+      data-testid="avatar-img"
+      src="data:image/png;base64,00"
```

Comment:
Unfortunately this does not assert the rendered colour. But at least the text content is different per test: `D`, `L` and `T`.